index.markdown

```diff
@@ -15,15 +15,15 @@ target="base-repo/index.html"
 background="assets/images/disks.jpg"
 title="base-repo"
 subtitle="A place to be for your data"
-content="The base-repo is a generic, general purpose research data repository service offering clear, machine-actionable RESTful interfaces for storing, retrieving and managing research data."
+content="The base-repo is a generic, general purpose research data repository service. It is offering clear, machine-actionable RESTful interfaces for storing, retrieving and managing research data."
 %}
 
 {% include card.html tags="metadata repository restful"
 target="metastore/index.html"
 background="assets/images/metadata.jpg"
 title="MetaStore"
-subtitle="Metadata - for sure"
-content="MetaStore is a research data repository service for managing formally quality-controlled metadata documents based on JSON or XML."
+subtitle="Storing consistent metadata"
+content="MetaStore is a research data repository service storing metadata documents and schemas. By associating each document with a schema and validating it, quality and consistency are maintained. Supports JSON and XML."
```

Comment:
Suggestions:
1. "... repository service for storing ..."
2. "... validating it, quality and consistency are ensured." Oder ganz umdrehen: "Quality and consistency are ensured by associating and validating each document against a schema." 
3. "It supports JSON and XML."

Reply:
I'd vote for 2+3. Thoughts regarding 1:

- It's ok, but I think a comma between service and storing would be enough.
- Service is such a general term and still sounds like a hosted/cloud solution. Maybe replace with software?

Reply:
I rephrased it and replaced service by software. See commit [d30e48b](https://github.com/kit-data-manager/webpage/pull/25/commits/d30e48bd68f611d8c2aa1ad061fdb863f0480212).